src/server.ts

```diff
@@ -0,0 +1,87 @@
+import express from 'express';
+import { randomUUID } from 'crypto';
+import bodyParser from 'body-parser';
+import { createValidator } from 'express-joi-validation';
+import {
+  createUserBodySchema,
+  paramsIDSchema,
+  paramsSubstringLimitSchema,
+  updateUserBodySchema,
+} from './schemas';
+import type { User } from './types';
+
+const app = express();
+const validator = createValidator();
+
+const users: User[] = [];
+
+app.use(bodyParser.json());
+app.use(express.json());
```

Comment:
You don't need both. I've just meant that you don't need to install one more package just to one middleware, as it's available at express as well. You can use bodyParser of course, if you want. Just one of them.

Reply:
fixed